things/api/things/http/requests.go

```diff
@@ -3,7 +3,9 @@
 
 package http
 
-import "github.com/mainflux/mainflux/things"
+import (
+	"github.com/mainflux/mainflux/things"
```

Comment:
this should be one line

Reply:
`gofmt` seems to prefer this method (multi-line parenthesis).  I switched it back as that is how it was originally.